web/admin/src/App.js

```diff
@@ -238,10 +242,48 @@ class App extends PureComponent {
             </ul>
           </div>
         </div>
+        <div className="csvLinkBox">
+          <button className="csvButton" onClick={this.formatDataForExport}>
+            Export Data
+          </button>
+          {this.state.isExporting ? (
+            <CSVDownload data={this.state.exportList} target="_blank" />
+          ) : null}
+        </div>
       </div>
     )
   }
 
+  formatDataForExport = () => {
+    const questions = Object.values(this.state.questions)
+    const exportList = questions.map(question => {
+      if (this.state.answersByQuestion[question.id]) {
+        const exportObject = {
+          creator: `${question.creator.firstName} ${question.creator.lastName}`,
+          Blocked: question.block ? 'Yes' : 'N/A',
+          Question: question.text,
+          Tags: question.filters,
+        }
+        Object.values(this.state.answersByQuestion[question.id]).forEach((item, i) => {
+          exportObject[`Response_${i + 1}`] = item.text
+          exportObject[`Response_${i + 1}_creator`] = `${item.creator.firstName} ${
+            item.creator.lastName
+          }`
+        })
+        return exportObject
+      }
+      return {
```

Comment:
This `return` is identical to the `exportObject` on line 261, except for the `Responses`.  Reduce code duplication by creating `exportObject` once (always, outside the line 260 `if`) and then conditionally add `Responses: 'None'` or the various response fields.